package.json

```diff
@@ -67,14 +68,17 @@
     "gulp-git": "^2.10.1",
     "husky": "^7.0.0",
     "identity-obj-proxy": "^3.0.0",
+    "image-js": "^0.33.0",
+    "jest": "^26.6.1",
```

Comment:
Could you explain why you added the `jest` dependency here?

Reply:
So, we're using `expect()` in `tests/e2e/helpers-native/expectToMatchImageSnapshot.ts` which requires to be of type`jest` framework and not of `jasmine`. As a result, we're adding jest dependency. Also, we had to remove some dependencies like  `"@types/jasmine": "^3.6.0"` for the same.

Reply:
`image-js` already contains the `jest` dependency. So I'm not sure if we actually need to add this dependency here. Did you try to run it without?

Reply:
Oh, didn't know about it. I tried without `jest` and it works absolutely fine. So, I've removed the jest dependency. I've also removed `"@types/jasmine": "^3.6.0"`, is that okay?

Reply:
I've added back `"@types/jasmine": "^3.6.0"` after discussing with @leonardomendix